Update README.m fixes #98 (#113)
shimming-toolbox/shimming-toolbox: README.md

# shimming-toolbox

> The shimming-toolbox is open-source Matlab software package enabling a variety of MRI shimming (magnetic field homogenization) techniques such as [static](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.25587) and [real-time](https://doi.org/10.1002/mrm.27089) shimming for use with standard manufacturer-supplied gradient/shim coils or with custom "multi-coil" arrays.

Reviewer: now we're talking 🚀

(create the file if it does not exist):

```
% Change ~/Code/shimming-toolbox/ to wherever the repository was downloaded/cloned:

> PATH_TO_REALTIME_SHIMMING-TOOLBOX = '~/Code/shimming-toolbox/' ;

Reviewer: i'm not a big fan of mixing _ and -
```suggestion
PATH_SHIMMINGTOOLBOX = '~/Code/shimming-toolbox/' ;
```

> addpath( genpath( PATH_TO_SHIMMING-TOOLBOX_REPOSITORY ) ) ;

Reviewer: ```suggestion
addpath( genpath( PATH_SHIMMINGTOOLBOX ) ) ;
```

```
# add MATLAB path

> export PATH=$PATH:/Applications/MATLAB_R2020a.app/bin/

Reviewer: this is specific to OSX-- we should probably be more general. Maybe just add a comment, specifying to point to matlab install

List of available maps:
- UNF: https://drive.google.com/drive/folders/1DSCTeh9qRCgS55fWLGtsNeYdq8DsRyMg

> Download map and copy it under $PATH_SHIMMING-TOOLBOX/data/

Reviewer: ```suggestion
Download map and copy it under $PATH_SHIMMINGTOOLBOX/data/
```
